### PR TITLE
Review Vignettes

### LINKED ISSUE #190: documentation: `cfun` accepts both df and x
Some places where cfun only mentions df are:

https://github.com/Roche/rtables/blob/5f6efc798d430d34479cc5992967d17c44ae4f99/R/argument_conventions.R#L47

https://github.com/Roche/rtables/blob/759eb771eca71be3e519b594aa76f6989fbf14fd/vignettes/clinical_trials.Rmd#L297

Also in the Details section of `summarize_row_groups`

### LINKED ISSUE #361: Review Vignettes II
Please read the following vignettes and try the examples. If you have improvement suggestions create a branch with a PR.

- [x] Example Clinical Trials Table
- [x] Comparing against baseline
- [x] Pruning and sorting tables

### PR DESCRIPTION
**Pull request by edelarua**
No updates needed for vignette examples - just some minor formatting/spelling/grammatical fixes for better readability.

Closes #361 

Also added missing `x` argument to `cfun` documentation where needed.

Closes #190

### CODE REVIEW

**Review by gmbecker — CHANGES_REQUESTED** [review]
Needs rebasing on main after merge of the function links PR. In the future please group PRs like this into larger chunks to avoid this issue.

File: R/colby_constructors.R [hunk 1]
```diff
@@ -1320,7 +1320,7 @@ setMethod(".add_row_summary", "Split",
 #'   column total counts are calculated. If `format` is a format string where
 #'   `xx` appears only one time, only raw counts are used.
 #'
-#' `cfun` must accept `df` as its first argument and will receive the subset
+#' `cfun` must accept `x` or `df` as its first argument and will receive the subset
```
[the remark this answers is not in the document]
Comment (edelarua) [reply R/colby_constructors.R]: I have reverted this change for now and will cover it as part of the new issue.

**Review by shajoezhu — APPROVED** [review]
lgtm! Thanks @edelarua

**Comment by shajoezhu** [comment]
Hi @gmbecker , I had another look at this. Looks improved. Just wondering what your thoughts are. Can we merge this already please? Thanks

**Review by gmbecker — APPROVED** [review]

**Pull request merged**